src/test/storage/table_test.cpp

```diff
@@ -68,4 +69,32 @@ TEST_F(StorageTableTest, GetColumnIdByName) {
 
 TEST_F(StorageTableTest, GetChunkSize) { EXPECT_EQ(t.target_chunk_size(), 2u); }
 
+TEST_F(StorageTableTest, ColumnNames) {
+  EXPECT_EQ(t.column_names()[0], "col_1");
+  EXPECT_EQ(t.column_names()[1], "col_2");
+}
+
+TEST_F(StorageTableTest, GetChunkConst) {
```

Comment:
According to coverage tool this doesn't actually cover the const getter 
![image](https://user-images.githubusercontent.com/34538290/118637682-5f054f80-b7d6-11eb-8977-bf985c2fe9c3.png)
One way to force const access would be `std::as_const`